### DISCUSSION TITLE
Opening new tab in foreground.

### DISCUSSION THREAD

**Discussion by pegasocv** [piroor/treestyletab#3182] [Q&A]
Hello,
I use Firefox and although I have set to always open a new tab in the foreground (even if I click a link) since I use TST this option does not work and the tab is opened in the background. 
I don't know English and it could be that I didn't find the right option, can anyone tell me if using TST it is possible to open all new tabs, even when clicking a link, in the foreground?
Thank you.

**Comment by irvinm** [comment]
So ... to be clear.  In Firefox you have this option set?

![image](https://user-images.githubusercontent.com/979729/188237693-7bd91a7f-2197-4ea0-ad0f-8b7f69c88585.png)

If you disable TST, it works as expected but when it is enabled it doesn't?  (I am not aware of any options in TST that changes this behavior.  Can you disable TST and verify that it works again?)

**Comment by pegasocv** [comment]
Yes I have that option enabled.
I disabled TST and the option worked, after enabling TST again the option still works.
Thank you.
